src/binder/alter_table.rs

```diff
@@ -0,0 +1,73 @@
+use sqlparser::ast::{AlterTableOperation, ObjectName};
+
+use std::sync::Arc;
+
+use super::Binder;
+use crate::binder::{lower_case_name, split_name, BindError};
+use crate::planner::operator::alter_table::{AddColumn, AlterTableOperator};
+use crate::planner::operator::scan::ScanOperator;
+use crate::planner::operator::Operator;
+use crate::planner::LogicalPlan;
+use crate::storage::Transaction;
+
+impl<'a, T: Transaction> Binder<'a, T> {
+    pub(crate) fn bind_alter_table(
+        &mut self,
+        name: &ObjectName,
+        operation: &AlterTableOperation,
+    ) -> Result<LogicalPlan, BindError> {
+        let table_name: Arc<String> = Arc::new(split_name(&lower_case_name(name))?.1.to_string());
+
+        // we need convert ColumnDef to ColumnCatalog
```

Comment:
is it what you need: https://github.com/KipData/KipSQL/blob/e7dbac15d7f8fac1e42b6b4291fc7251e2685d88/src/binder/create_table.rs#L85

Reply:
It looks like it has been used below, so this comment is probably unnecessary?

Reply:
this annotation is deprecated, I forget remove it